apps/system/js/app_window.js

```diff
@@ -2514,19 +2514,14 @@
    * @param {Object?} siteObj
    * @returns {Promise}
    */
-  AppWindow.prototype.getIconBlob = function ac_getIconBlob(origin, iconSize,
+  AppWindow.prototype.getIcon = function ac_getIcon(origin, iconSize,
     placeObj = {}, siteObj = {}) {
 
     return new Promise((resolve, reject) => {
       LazyLoader.load('/shared/js/icons_helper.js').then(() => {
-        IconsHelper.getIconBlob(origin, iconSize, placeObj, siteObj)
```

Comment:
So, if we don't use `IconsHelper.getIconBlob` anymore, how do we manage to store the icons in the `icons` dataStore?